lib/parse-args.js

```diff
@@ -154,6 +170,10 @@ function buildYargs (withCommands = false) {
     })
     .pkgConf('c8')
     .demandCommand(1)
+    // Todo:  These next 6 lines might need to get removed
+    // This logic is covered with a default argument in
+    // the config options declaration in this file for yargs
+    /* c8 ignore next 6 */
     .check((argv) => {
       if (!argv.tempDirectory) {
         argv.tempDirectory = resolve(argv.reportsDir, 'tmp')
```

Comment:
From my understanding, this function should not actually change arguments but only checks to see if they are valid, per the [yargs documentation.](https://yargs.js.org/docs/#api-reference-checkfn-globaltrue)

This condition was never reached per the former code coverage report.  Added a coverage ignore comment line. 

Setting the tempDirectory when empty should be addressed in this patch on [lines 31 - 42](https://github.com/bcoe/c8/pull/447/files/0eef52ad5780e954315eb37c12235603f498a35c#diff-40d93e4e5f4b94fc8a34a87a3328651c24e855051bfcdd6944dee76f9c82b8d4R31-R42).

I suggest these lines can be deleted.

Reply:
@bcoe I believe this is the last action item to be resolved before opening up the workflow.  At the moment it has a coverage ignore comment block and removing the block doesn't affect the test results.  I am almost certain that this block of code is covered in [lines 19 - 26](https://github.com/bcoe/c8/pull/447/commits/692ec75f628fddc426ddda6a09f2a7ca95240611#diff-40d93e4e5f4b94fc8a34a87a3328651c24e855051bfcdd6944dee76f9c82b8d4R19-R26) and [line 149](https://github.com/bcoe/c8/pull/447/commits/692ec75f628fddc426ddda6a09f2a7ca95240611#diff-40d93e4e5f4b94fc8a34a87a3328651c24e855051bfcdd6944dee76f9c82b8d4R149).  Thoughts?

Reply:
> From my understanding, this function should not actually change arguments but only checks to see if they are valid

It might be less confusing to `corece` the temp directory, but we should only run the `resolve` logic if temp directory isn't set, to match the existing logic. And we shouldn't do any coercion on the report directory.